### PR TITLE
fix things affected by the renames

### PR DESCRIPTION
**Pull request by fluffypony**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,18 +23,18 @@ RUN set -e && \
   apt-get clean -q -y && \
   apt-get autoclean -q -y && \
   apt-get autoremove -q -y && \
-  mkdir /bitmonero && \
-  mv /src/build/release/bin/* /bitmonero && \
+  mkdir /monero && \
+  mv /src/build/release/bin/* /monero && \
```
Comment (moneromooo-monero) [inline Dockerfile]: Does docker really write to / ?

File: Dockerfile [hunk 2]
```diff
@@ -23,18 +23,18 @@ RUN set -e && \
...
   rm -rf /src
 
-WORKDIR /bitmonero
+WORKDIR /monero
 
 # Contains the blockchain
 VOLUME /root/.bitmonero
```
Comment (moneromooo-monero) [inline Dockerfile]: This was not changed (to /root/.monero) ?

File: README.md [hunk 3]
```diff
@@ -2,15 +2,15 @@
 
 Copyright (c) 2014-2016, The Monero Project
 
-[![Build Status](https://travis-ci.org/monero-project/bitmonero.svg?branch=master)](https://travis-ci.org/monero-project/bitmonero)
-[![Coverage Status](https://coveralls.io/repos/github/monero-project/bitmonero/badge.svg?branch=master)](https://coveralls.io/github/monero-project/bitmonero?branch=master)
+[![Build Status](https://travis-ci.org/monero-project/monero.svg?branch=master)](https://travis-ci.org/monero-project/monero)
+[![Coverage Status](https://coveralls.io/repos/github/monero-project/monero/badge.svg?branch=master)](https://coveralls.io/github/monero-project/monero?branch=master)
```
Comment (moneromooo-monero) [inline README.md]: Did this actually change yet ?

File: src/blockchain_utilities/README.md [hunk 4]
```diff
@@ -100,21 +83,11 @@ BerkeleyDB flags (takes one):
 
 ```
 ## Examples:
-$ blockchain_import --database lmdb#fastest
-$ blockchain_import --database berkeley#fastest
+$ monero-blockchain-import --database lmdb#fastest
+$ monero-blockchain-import --database berkeley#fastest
```
Comment (moneromooo-monero) [inline src/blockchain_utilities/README.md]: I think we can remove examples for bdb, I don't think it'd actually work anymore

File: src/cryptonote_core/cryptonote_core.cpp [hunk 5]
```diff
@@ -280,8 +280,8 @@ namespace cryptonote
       {
         LOG_PRINT_RED_L0("Found old-style blockchain.bin in " << old_files.string());
         LOG_PRINT_RED_L0("Monero now uses a new format. You can either remove blockchain.bin to start syncing");
-        LOG_PRINT_RED_L0("the blockchain anew, or use blockchain_export and blockchain_import to convert your");
-        LOG_PRINT_RED_L0("existing blockchain.bin to the new format. See README.md for instructions.");
+        LOG_PRINT_RED_L0("the blockchain anew, or use monero-blockchain-import and monero-blockchain-export to");
+        LOG_PRINT_RED_L0("convert your existing blockchain.bin to the new format. See README.md for instructions.");
```
Comment (moneromooo-monero) [inline src/cryptonote_core/cryptonote_core.cpp]: Pedantic, but you probably meant to keep the export/import ordering

**Comment by moneromooo-monero** [comment]
ok modulo comments

File: README.md (hunk 3 above)
Comment (fluffypony) [inline README.md]: Yes - Travis automatically picked it up, Coveralls needed a "sync from Github" prod

File: src/blockchain_utilities/README.md (hunk 4 above)
Comment (fluffypony) [inline src/blockchain_utilities/README.md]: I was going to ask about that as a thing-to-do-next, I couldn't remember

File: src/cryptonote_core/cryptonote_core.cpp (hunk 5 above)
Comment (fluffypony) [inline src/cryptonote_core/cryptonote_core.cpp]: I did, my bad

**Comment by moneromooo-monero** [comment]
ok

**Pull request merged**